£Lerror_reason¢DtypeEvalueXûHTTPConnectionPool(host='chime.ics.uci.edu', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000000000456B518>, 'Connection to chime.ics.uci.edu timed out. (connect timeout=2)'))Ihttp_code¢DtypeEvalueóQdownload_complete¢DtypeEvalueõ